src/main/java/org/mskcc/cbio/oncokb/web/rest/errors/DatabaseReadOnlyException.java

```diff
@@ -0,0 +1,13 @@
+package org.mskcc.cbio.oncokb.web.rest.errors;
+
+import org.zalando.problem.AbstractThrowableProblem;
+import org.zalando.problem.Status;
+
+public class DatabaseReadOnlyException extends AbstractThrowableProblem{
+
+    private static final long serialVersionUID = 1L;
+
+    public DatabaseReadOnlyException() {
+        super(ErrorConstants.SERVICE_UNAVAILABLE, "Service is temporarily unavailable", Status.SERVICE_UNAVAILABLE);
```

Comment:
Let's be clear here. Maybe `We are not able to update the content at this moment. Service should be back momentarily.`?